src/renderer/assets/css/_spacing.sass

```diff
@@ -17,3 +17,19 @@
 @for $i from 1 through 10
 	.m-#{$i}
 		margin: $i + rem
+
+@for $i from 1 through 10
+	.mt-#{$i}
+		margin-top: $i + rem
+
+@for $i from 1 through 10
+	.ml-#{$i}
+		margin-left: $i + rem
+
+@for $i from 1 through 10
+	.mb-#{$i}
+		margin-bottom: $i + rem
+
+@for $i from 1 through 10
+	.mr-#{$i}
+		margin-right: $i + rem
```

Comment:
Creating for loop for margins might take a lot of space in terms of file size and we can easily add those to class based css inside components files.